moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Box, Grid, Typography } from "@mui/material";
+
+/**
+ * ProjectSummaryCouncilDistricts Component
+ * @param {Object} data - The project geography from the graphql query's data object
+ * @param {Object} classes - The shared style settings
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryCouncilDistricts = ({ projectGeography, classes }) => {
+  // reduce the array of geography objects into an array of city council districts
+  const getDistricts = (data) => {
+    const initialValue = [];
+    const districts = data.reduce(
+      (acc, component) => [...acc, component["council_districts"]],
+      initialValue
+    );
+
+    return [...new Set(districts.flat())];
+  };
```

Comment:
you can move this function outside the component so that it is not re-assigned on every rendered. i don't know if it is worthwhile to memoize it 🤷